components/lib/dropdown/Dropdown.js

```diff
@@ -352,22 +355,26 @@ export const Dropdown = React.memo(
             return selectedIndex < 0 ? findLastOptionIndex() : selectedIndex;
         };
 
+
+        const appropriateOptions = () => flatedGroupOptions ?? visibleOptions
```

Comment:
maybe its better to use a simple variable

Reply:
Yes move this up to where you get `flattenedOptions`  and shouldn't `visibleOptions = flattenedOptions` and then all this code will just work using `visibleOptions`?

Reply:
visibleOptions need to be the same as it was, this format will be used at DropdownPanel component to make groups

Reply:
I cant  access visibleOptions  before initialization, should i leave it as method or move up their initialization ?